internal/provider/credential_provider_resource.go

```diff
@@ -192,6 +192,10 @@ func (r *credentialProviderResource) Schema(_ context.Context, _ resource.Schema
 						Description: "Scopes for the OAuth Credential Provider.",
 						Optional:    true,
 					},
+					"credential_style": schema.StringAttribute{
+						Description: "Credential Style for the OAuth Credential Provider.",
+						Optional:    true,
```

Comment:
Can you add a default value here set to authHeader please?
This will simplify migration from old to new

Reply:
I added the default value